openshift-install-powervs

```diff
@@ -593,8 +594,17 @@ function precheck_input {
     if [ ! -f "var.tfvars" ]; then
       warn "No variables specified or var.tfvars does not exist.. running variables command" && variables
     fi
-    varfile="var.tfvars"
-    vars="-var-file ../$varfile"
+
+    for((var = 1; var <= $#; var++)); do
```

Comment:
I do not understand the purpose of this loop. You are already appending compute-vars in main function under -flavor case.

Reply:
This loop checks if the command contains a -flavor argument. If so, it can pick the flavor specified and can replace it when the varfile is used containing the default flavor. This should resolve 
`What if I give -flavor option before -varfile ? I feel varfile will be "".`